### PR TITLE
Insertionsort

### PR DESCRIPTION
**Pull request by littl3birdy**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/InsertionSort.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package code401challenges;
+
+public class InsertionSort {
+    public void insertSort(int[] arr){
```
Comment (SharinaS) [inline code401challenges/src/main/java/code401challenges/InsertionSort.java]: To run your tests the way you've written them, you can change change your method signature to a static:
```
public static void insertSort(int[] arr) { }
```

File: code401challenges/src/test/java/code401challenges/InsertionSortTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import static org.junit.Assert.*;
+
+public class InsertionSortTest {
+
+    @Test
+    public void testInsertSort(int[] testArray) {
+        int[] testArray = new int[]{8,4,23,42,16,15};
+        int [] expected = new int[]{4,8,15,16,23,42};
+        assertArrayEquals(expected, insertSort(testArray));
```
Comment (SharinaS) [inline code401challenges/src/test/java/code401challenges/InsertionSortTest.java]: You need to tell Java where the method lives that you're calling, so if you're calling insertSort, you need to reference the class it comes from, and then give it an array to work on. Do this before line 13. 
``` 
InsertionSort.insertSort(testArray);
```
Now, your assert statement has something to evaluate.
